recipes/cmdstan/build.sh

```diff
@@ -0,0 +1,32 @@
+mkdir -p $PREFIX/bin
+cp -r . $PREFIX/bin/cmdstan
+
+cd $PREFIX/bin/cmdstan
+
+echo "TBB_CXX_TYPE=${c_compiler}"  >> make/local
+#echo "TBB_INTERFACE_NEW=true" >> make/local
+echo "TBB_INC=${PREFIX}/include/" >> make/local
+echo "TBB_LIB=${PREFIX}/lib/" >> make/local
+echo "PRECOMPILED_HEADERS=false" >> make/local
+
+cat make/local
+
+make clean-all
+
+make build -j${CPU_COUNT}
+
+# set up an alias. see https://github.com/stan-dev/cmdstan/issues/1024
+echo "#!/bin/bash" > "${PREFIX}/bin/cmdstan_model"
+echo "fname=\${1%.stan}" >> "${PREFIX}/bin/cmdstan_model"
+echo "make -C ${PREFIX}/bin/cmdstan \$(realpath --relative-to=${PREFIX}/bin/cmdstan \$fname)" >> "${PREFIX}/bin/cmdstan_model"
+chmod +x "${PREFIX}/bin/cmdstan_model"
+
+# activate script
+echo "export CMDSTAN=${PREFIX}/bin/cmdstan" > "${RECIPE_DIR}/activate.sh"
```

Comment:
Activate script should backup any existing `CMDSTAN` variable and deactivate should restore the backup. Btw, why is `CMDSTAN` needed?

Reply:
`CMDSTAN` is used by the Python and R bindings (‘CmdStanPy’ and ‘CmdStanR’) to find CmdStan if it is not installed in the users home directory. We set it here so that this package is plug and play with those bindings, which we believe will be the majority of the install base based on current usage

I will make the change to set a temporary and restore it

Reply:
`CMDSTAN` is now stored to `CMDSTAN_OLD` by activate, and restored to that value in deactivate